src/components/DetailedPodcast/DetailedPodcast.js

```diff
@@ -0,0 +1,98 @@
+import FavorizeButton from '../FavorizeButton/FavorizeButton'
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+DetailedPodcast.propTypes = {
+  podcast: PropTypes.object,
+}
+
+export default function DetailedPodcast({ podcast }) {
+  return (
+    <ContentWrapper>
+      <ImageContainer>
+        <Image src={podcast.image} alt={`Poster`} width="40px" height="40px" />
```

Comment:
If you don't pass the unit (px) it gets transformed to aspect ratio & blocks therefore layout shifts. Since you style it anyway this might what you wanted to achieve here

Reply:
So you mean pass in the widths and heights without units?